portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -35,10 +38,17 @@
 public class DataServlet extends HttpServlet {
   @Override
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    String currComment = request.getParameter("comment");
-    writeComments(currComment);
+    String originalText = request.getParameter("text");
+    String languageCode = request.getParameter("languageCode");
+    Translate translate = TranslateOptions.getDefaultInstance().getService();
+    Translation translation =
```

Comment:
Can translate be a class variable?

Reply:
causes compilation error when I make translate a class variable